beacon_node/network/src/beacon_processor/work_reprocessing_queue.rs

```diff
@@ -117,13 +141,16 @@ struct ReprocessQueue<T: BeaconChainTypes> {
 
     /* Queues */
     /// Queue to manage scheduled early blocks.
-    block_delay_queue: DelayQueue<QueuedBlock<T>>,
+    gossip_block_delay_queue: DelayQueue<QueuedGossipBlock<T>>,
+    /// Queue to manage scheduled early blocks.
+    rpc_block_delay_queue: DelayQueue<QueuedRpcBlock<T::EthSpec>>,
     /// Queue to manage scheduled attestations.
     attestations_delay_queue: DelayQueue<QueuedAttestationId>,
 
     /* Queued items */
     /// Queued blocks.
-    queued_block_roots: HashSet<Hash256>,
+    queued_gossip_block_roots: HashSet<Hash256>,
+    queued_rpc_block_roots: HashSet<Hash256>,
```

Comment:
thinking on the fact we might get duplicate blocks for RPC works, I think we don't really need this

Reply:
Ohh yeah good point. We get rpc requests only from sync and it should never make duplicate requests. I'll remove this.